tasks/post_copy_request_to_queue/post_copy_request_to_queue.py

```diff
@@ -56,44 +58,75 @@ def lambda_handler(event, context):
         AND
             status_id = %d
     """
-    #Gets the dbconnection info.
+    # Gets the dbconnection info.
 
-    db_connect_info = requests_db.get_dbconnect_info()
+    db_connect_info = get_dbconnect_info()
 
     try:
-        rows = database.single_query(sql, database.db_connect_info, (filename, shared_recovery.OrcaStatus.PENDING))
-    except database.DbError as err:
-        LOGGER.error(f"DbError: {str(err)}")
-        raise DatabaseError(str(err))
+        rows = single_query(sql, db_connect_info, (filename, OrcaStatus.PENDING.value))
+    except Exception as ex:
+        LOGGER.error(ex)
 
     if len(rows) == 0:
-        return None
-    orca_recoveryfile = rows[0]
-    # grab the parameters from the db in json format
-    job_id = database.result_to_json(orca_recoveryjob)['job_id']
-    granule_id = database.result_to_json(orca_recoveryjob)['granule_id']
-    restore_destination = database.result_to_json(orca_recoveryjob)['restore_destination']
-
-    #post to staged-recovery-queue
-    shared_recovery.post_status_for_file_to_queue(
-        job_id = job_id,
-        granule_id = granule_id,
-        filename = filename,
-        restore_destination = restore_destination,
-        status_id = shared_recovery.OrcaStatus.STAGED,
-        error_message = '',
-        request_method = shared_recovery.RequestMethod.NEW,
-        db_queue_url= OS_ENVIRON_RECOVERY_QUEUE_URL_KEY
+        raise Exception(
+            "length of rows cannot be empty since there will always be a record in the database"
         )
+    # convert db result to json
+    db_result_json = result_to_json(rows)
+
+    # grab the parameters from the db in json format
+    job_id = db_result_json["job_id"]
+    granule_id = db_result_json["granule_id"]
+    restore_destination = db_result_json["restore_destination"]
+
+    # post to staged-recovery-queue
+    post_status_for_file_to_queue(
+        job_id,
+        granule_id,
+        filename,
+        restore_destination,
+        OrcaStatus.STAGED.value,
+        "",
+        RequestMethod.NEW.value,
+        RECOVERY_QUEUE_URL,
+    )
 
-    #post to DB-queue
-    shared_recovery.post_status_for_file_to_queue(
-        job_id = job_id,
-        granule_id = granule_id,
-        filename = filename,
-        restore_destination = restore_destination,
-        status_id = shared_recovery.OrcaStatus.STAGED,
-        error_message = '',
-        request_method = shared_recovery.RequestMethod.NEW,
-        db_queue_url= OS_ENVIRON_DB_QUEUE_URL_KEY
-        )
\ No newline at end of file
+    # post to DB-queue
```

Comment:
You may want to put this in a try block to catch exceptions, log them and retry a certain number of times.